string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,13 @@
+/* exported capitalizeWord */
+
+/* its the same as the capitalize function from level one but you have to add a
+condition to deal with the word "JavaScript" specifically
+*/
+
+function capitalizeWord(string) {
+  if (string.toLowerCase() === 'javascript') {
+    return 'JavaScript';
+  } else if (string.charAt(0)) {
```

Comment:
What is the condition here meant to represent? I think this can just be an `else`